vspace/src/main/java/edu/asu/diging/vspace/web/staff/StaffSearchModuleController.java

```diff
@@ -0,0 +1,85 @@
+package edu.asu.diging.vspace.web.staff;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.diging.vspace.core.model.ISlide;
+import edu.asu.diging.vspace.core.model.impl.Module;
+import edu.asu.diging.vspace.core.model.impl.Slide;
+import edu.asu.diging.vspace.core.model.impl.StaffSearchModule;
+import edu.asu.diging.vspace.core.services.ISequenceManager;
+import edu.asu.diging.vspace.core.services.IStaffSearchManager;
+
+@Controller
+public class StaffSearchModuleController {
+
+    @Autowired
+    private IStaffSearchManager staffSearchManager;
+
+    @Autowired
+    private ISequenceManager sequenceManager;
+
+    @RequestMapping(value = "/staff/search/module")
+    public ResponseEntity<StaffSearchModule> searchInVspace(
+            @RequestParam(value = "modulePagenum", required = false, defaultValue = "1") String modulePagenum,
+            Model model, @RequestParam(name = "searchText") String searchTerm) {
+
+        List<Module> moduleList = paginationForModule(modulePagenum, searchTerm);
+        StaffSearchModule staffSearch = new StaffSearchModule();
+        staffSearch.setModules(moduleList);
+
+        Map<String, String> moduleFirstSlideImage = new HashMap<>();
+        Map<String, Boolean> isModuleConfiguredMap = new HashMap<>();
+
+        for (Module module : moduleList) {
+
+            if (module.getStartSequence() == null) {
+                isModuleConfiguredMap.put(module.getId(), true);
```

Comment:
if there is no start sequence, the module is not configured. Why would "isModuleConfigured" be true? It would be false.